utils/zerovec/derive/src/ule.rs

```diff
@@ -278,10 +299,8 @@ fn make_ule_enum_impl(
 }
```

Comment:
Question: Should the to/from `u8` impls be optional, if they conflict with something else?

Reply:
We should at least make it more visible and explicit that `#[make_ule]` on an enum generates these impls.

Reply:
What if we make inherent methods instead?

Reply:
Should we consider a default method on AsULE that converts a single element rather than a slice? That might be generally useful anyway.

Reply:
Maybe yeah.

Reply:
Thinking about it more, I think an inherent method isn't that necessary, and we can always make it optional later.

Reply:
A default method on ULE won't be useful here anyway because that converts between the ULE type and the byte slice, not to the inner integer. I think we should leave this as is for now.

Reply:
I'm fine making this optional in the future if the impls collide with something.